rest_api/custom_component.py

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
How about moving this file into a new folder `rest_api/pipelines` together with the yml (and possibly multiple ymls coming up in #1158?

I would generally also prefer naming it "custom_nodes.py" as nodes as I think that's more intuitive and closer to how we explain it to users. However, as we are currently calling it components in the YML we can stick with components for now and discuss / tackle the renaming in a separate PR

Reply:
Ah, one other requirement just came into my mind: would be nice if users can mount custom pipelines / nodes into a docker container. So another reason to have a separate folder that they can overwrite at runtime without losing the rest API code etc.